Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM ubuntu:20.04
+
+USER root
+
+RUN apt-get update && apt-get upgrade -y
+
+RUN apt install -y python3 python3-pip
+
+# utils
+RUN apt install -y wget git
+
+# golang
+RUN wget https://dl.google.com/go/go1.17.7.linux-amd64.tar.gz && \
+  tar -xzvf go1.17.7.linux-amd64.tar.gz -C /usr/local && \
+  export PATH=$PATH:/usr/local/go/bin && echo 'export PATH=$PATH:/usr/local/go/bin' >> ~/.bashrc && \
+  mkdir $HOME/go
+
+ENV PATH="${PATH}:/usr/local/go/bin"
```

Comment:
Why not start from a golang image?

Reply:
we could slim the image down, not in this PR though